src/client/java/teammates/client/scripts/sql/VerifyCourseEntityCounts.java

```diff
@@ -54,6 +57,11 @@ private Long countPostgresEntities(Class<? extends teammates.storage.sqlentity.B
 
     @Override
     protected void doOperation() {
+        verifyCurrentEntities();
```

Comment:
Maybe add a javadoc comment to explain this, I can't think of a better name either